style.css

```diff
@@ -0,0 +1,136 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;1,700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+
+/* Reset */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 65.5%;
+  display: flex;
+  flex-direction: column;
+}
+
+body {
+  line-height: 1;
+  overflow-x: hidden;
+}
+
+.padding-left-right {
+  padding: 0 2.4rem;
+}
+
+.header {
+  display: flex;
+  grid-template-columns: 1fr auto;
+  align-items: center;
+  height: 4.8rem;
+  background-color: #3c3a39;
+  color: #fff;
+}
+
+.header a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.logo {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700rem;
+  font-size: 1.8rem;
+  line-height: 2.4rem;
+}
+
+.headline-section {
+  display: flex;
+  flex-direction: column;
+  background-color: #1c1a19;
+  color: #fff5e1;
+  padding-top: 12rem;
+  padding-bottom: 18rem;
+  position: relative;
+}
+
+.headline-box {
+  display: flex;
+  flex-direction: column;
+}
+
+.headline-text-box {
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
+  color: #fff5e1;
+}
+
+#headline-primary {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 5.6rem;
+  line-height: 6.4rem;
+  letter-spacing: 0.037rem;
+  color: #f76300;
+}
+
+.headline-title {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2.4rem;
+  line-height: 3.2rem;
+  color: #fff5e1;
+}
+
+.headline-description {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1.6rem;
+  line-height: 2.4rem;
+  color: #fff;
+  margin-bottom: 2.4rem;
+}
+
+.icon {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.social-icons-list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.social-icons-list nav {
```

Comment:
- [ ] Please I highly recommend sticking to the required [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=HQA60gDOdof1M29t-0). 😄 

_**REQUIRED FIGMA DESIGN**_

![Screenshot (240)](https://user-images.githubusercontent.com/78104544/235161042-ba451664-bd30-4574-a2fe-f000e0310889.png)


_**YOUR DESIGN**_

- [ ] Please I highly recommend making changes to the `social-icons-lists` class to fit the display of the required figma design as requested by the previous reviewer. 😄 
- [ ] Please I suggest adding `social-icons-lists` class to the `style.css` file and add `display: flex` property to the class to fit the required figma design. 😄 

![Screenshot (241)](https://user-images.githubusercontent.com/78104544/235161143-79e2db8a-2f75-4377-a5a5-1b2af8c8c192.png)